planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/README.md

```diff
@@ -20,21 +20,21 @@ So for example, in order to stop at a stop line with the vehicles' front on the
 
 ## Input topics
 
-| Name                          | Type                                             | Description          |
-| ----------------------------- | ------------------------------------------------ | -------------------- |
-| `~input/path_with_lane_id`    | autoware_planning_msgs::PathWithLaneId           | path with lane_id    |
-| `~input/vector_map`           | autoware_lanelet2_msgs::MapBin                   | vector map           |
-| `~input/vehicle_velocity`     | geometry_msgs::TwistStamped                      | vehicle velocity     |
-| `~input/dynamic_objects`      | autoware_perception_msgs::DynamicObjectArray     | dynamic objects      |
-| `~input/no_ground_pointcloud` | sensor_msgs::PointCloud2                         | obstacle pointcloud  |
-| `~input/traffic_light_states` | autoware_perception_msgs::TrafficLightStateArray | traffic light states |
+| Name                          | Type                                              | Description          |
+| ----------------------------- | ------------------------------------------------- | -------------------- |
+| `~input/path_with_lane_id`    | autoware_auto_planning_msgs::PathWithLaneId       | path with lane_id    |
+| `~input/vector_map`           | autoware_auto_mapping_msgs::HADMapBin             | vector map           |
+| `~input/vehicle_velocity`     | geometry_msgs::TwistStamped                       | vehicle velocity     |
```

Comment:
```suggestion
| `~input/vehicle_velocity`     | nav_msgs::Odometry                       | vehicle velocity     |
```

Reply:
Sorry, I did that in this commit
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/605/commits/d42b4c4a79dbcb5c123d6944a9f7cff89311a206